hivemind/p2p/p2p_daemon.py

```diff
@@ -422,15 +434,38 @@ async def _stream_handler(requests: P2P.TInputStream, context: P2PContext) -> P2
 
         await self._add_protobuf_stream_handler(name, _stream_handler, input_protobuf_type)
 
+    # only registers request-response handlers
```

Comment:
nit: Move this to docstring and also explain that the request-response handlers are processed differently for the sake of optimization of performance/number of open files.

Reply:
Let's remove the comment since we now have the docstring :)

```suggestion
```